src/core/sock.c

```diff
@@ -48,6 +49,7 @@
 #define NN_SOCK_STATE_ZOMBIE 3
 #define NN_SOCK_STATE_STOPPING_EPS 4
 #define NN_SOCK_STATE_STOPPING 5
+#define NN_SOCK_STATE_FINI 6
```

Comment:
FINI - "finish".  I could have use "STOPPED" here equally, although STOPPED has some other uses which may be confusing in this file.